Kyrychenko/quizEntity/src/main/java/ui/QuizServlet.java

```diff
@@ -0,0 +1,61 @@
+package ui;
+
+import model.Quiz;
+import service.QuizService;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Arrays;
+
+
+public class QuizServlet extends HttpServlet {
+    private static QuizService quizService = QuizService.getInstance();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String action = req.getParameter("action");
+
+        switch (action == null ? "add" : action) {
+            case "add":
+                doPost(req,resp);
```

Comment:
why we need to go to post through get?

Reply:
Uploaded new realization. This case have been changed.